DRODLibTests/src/Runner.cpp

```diff
@@ -107,7 +107,7 @@ void Runner::InitializeDatPath(){
 	wstrDatPathTxt += wstrExtension;
 
 	pErrorLogPath = new char[wstrDatPathTxt.length() * 2];
-	UnicodeToUTF8(wstrDatPathTxt.c_str(), pErrorLogPath);
+	UnicodeToAscii(wstrDatPathTxt.c_str(), pErrorLogPath);
```

Comment:
Thank you for correcting my regression here.

Would you please share how my change broke things?  I would not have expected this change to cause any issues.

Reply:
`pErrorLogPath` is a an array of `char`s, and there seems to be no overload for `UnicodeToUTF8` that converts the text to `char`s. And I need chars because i use `stat()` which accepts that argument.